_slides/13-whsieh-lac.md

```diff
@@ -3,7 +3,7 @@ layout: slide
 title: "Welcome to our slide deck!"
 ---
 
-Caption here
+Here's Wilson's Caption
```

Comment:
Hey, good caption.

Can we **consider** inserting your user name in the _caption_.

Reply:
```suggestion
Here's Wilson Hsieh's Caption
```